cmd/client/util.go

```diff
@@ -125,74 +125,70 @@ func parseAttributes(rootArgs *rootArgs) (*mixerpb.Attributes, error) {
 
 	// the following boilerplate would be more succinct with generics...
 
-	if m, err := process(attrs.Dictionary, rootArgs.stringAttributes, parseString); err != nil {
+	var m map[int32]interface{}
+	var err error
+
+	if m, err = process(attrs.Dictionary, rootArgs.stringAttributes, parseString); err != nil {
```

Comment:
Yes these changes were motivated by the vetshadow linter complaining about shadow definitions.

I don't have a log to link to, this was just discovered by running 'make lint' on my system. I'll try to integrate into Travis once I get the warnings tackled.